recipes/phyloacc/meta.yaml

```diff
@@ -32,15 +32,16 @@ requirements:
   host:
     - gsl
     - openblas
+    - blis
     - liblapack
     - armadillo
-    - python
+    - python>=3.7
```

Comment:
Please remove `>=3.7` and change the `skip: true` online 25 to `skip: True  # [py < 37]`

Reply:
Changes made in https://github.com/bioconda/bioconda-recipes/pull/34107/commits/80b20488e3a55493706d1409a4805d4b903c4bb5. Actual change is `skip: True # [py < 38]` because of the numpy version.